shell/platform/tizen/tizen_input_method_context.cc

```diff
@@ -191,6 +197,51 @@ bool TizenInputMethodContext::HandleEvasEventKeyUp(Evas_Event_Key_Up* event) {
       reinterpret_cast<Ecore_IMF_Event*>(&imf_event));
 }
 
+#ifdef NUI_SUPPORT
+bool TizenInputMethodContext::HandleNuiKeyEvent(const char* device_name,
+                                                uint32_t device_class,
+                                                uint32_t device_subclass,
+                                                const char* key,
+                                                const char* string,
+                                                uint32_t modifiers,
+                                                uint32_t scan_code,
+                                                size_t timestamp,
+                                                bool is_down) {
+  Ecore_Event_Key event;
+  event.keyname = event.key = key ? key : "";
+  event.string = string ? string : "";
+  event.modifiers = modifiers;
```

Comment:
Does `modifiers` represent "Ecore" modifiers?

Reply:
As far as I know this is Ecore_Event because it comes from Dali.